FinnUI/Sources/Fullscreen/NotificationCenter/Views/PersonalNotificationIconView.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  Copyright © 2020 FINN.no AS. All rights reserved.
+//
+
+import UIKit
+
+public final class PersonalNotificationIconView: UIView {
+    
+    public enum Kind {
+        case favorite
+        case myAds
+    }
+    
+    private lazy var imageView = UIImageView(
+        withAutoLayout: true
+    )
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        layer.cornerRadius = 12
+        layer.shadowOpacity = 0.3
+        layer.shadowOffset = CGSize(width: 0, height: 1)
+        layer.shadowRadius = 1.2
+        layer.borderColor = .milk
+        layer.borderWidth = 2
+        
+        imageView.tintColor = .milk
+        addSubview(imageView)
+        
+        NSLayoutConstraint.activate([
+            imageView.centerXAnchor.constraint(equalTo: centerXAnchor),
+            imageView.centerYAnchor.constraint(equalTo: centerYAnchor),
+            imageView.widthAnchor.constraint(equalToConstant: 12),
+            imageView.heightAnchor.constraint(equalToConstant: 12)
+        ])
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    func configure(with icon: Kind?) {
+        switch icon {
+        case .favorite:
+            backgroundColor = .secondaryBlue
+            imageView.image = UIImage(named: .heartMini)
+        case .myAds:
+            backgroundColor = .primaryBlue
```

Comment:
Is it on purpose that the dark mode compatible colors are not used in this view?

Reply:
Yes, this view represents the icon as a whole and the icons have the same background regardless of dark mode 😉 They use some new colors as well

Reply:
Ok, so darkmode-compatible without changing colors then I suppose 🤔